docs/wallet-extension/wallet-extension.md

```diff
@@ -30,13 +30,7 @@ The diagram below lays out the flows.
 
 ## Usage
 
-1. You have two choices for getting the wallet extension:
-
-   a. Download the wallet extension binary for your operating system from the list of Assets in the [releases area on GitHub](https://github.com/obscuronet/go-obscuro/releases).
-   
-   Or
-
-   b. Compile the wallet extension binary by cloning the [Obscuro repository](https://github.com/obscuronet/go-obscuro)
+1. Compile the wallet extension binary by cloning the [Obscuro repository](https://github.com/obscuronet/go-obscuro)
```

Comment:
Why did you remove the download option? Its less friction.

Reply:
James and Joel asked for it to be removed